package.json

```diff
@@ -2,10 +2,10 @@
   "name": "revolv",
   "version": "1.0.0",
   "devDependencies": {
-    "node-sass": "~0.7.0",
     "grunt": "~0.4.1",
     "grunt-contrib-watch": "~0.5.3",
-    "grunt-sass": "~0.8.0"
+    "grunt-sass": "^0.18.1",
+    "node-sass": "~0.7.0"
```

Comment:
Did you actually end up needing node sass in the devDependencies? If not, we should probably take it out, since sass is installed via rubygems on the VM